test/types/community/slow-model.test-d.ts

```diff
@@ -0,0 +1,419 @@
+/*
+ * The following type testing was adapted from @ermik
```

Comment:
should we create a release 4.11.0 ticket and call it out there so that we make sure we give credit in our release notes as well?

Reply:
https://jira.mongodb.org/browse/NODE-4718